website/docs/r/iothub.html.markdown

```diff
@@ -190,13 +190,13 @@ An `endpoint` block supports the following:
 
 * `name` - (Required) The name of the endpoint. The name must be unique across endpoint types. The following names are reserved:  `events`, `operationsMonitoringEvents`, `fileNotifications` and `$default`.
 
-* `authentication_type` - (Optional) Type used to authenticate against the endpoint. Possible values are `keyBased` and `identityBased`. Defaults to `keyBased`.
+* `authentication_type` - (Optional) The type used to authenticate against the endpoint. Possible values are `keyBased` and `identityBased`. Defaults to `keyBased`.
 
-* `identity_id` - (Optional) ID of the User Managed Identity used to authenticate against the endpoint.
+* `identity_id` - (Optional) The ID of the User Managed Identity used to authenticate against the endpoint.
 
--> **NOTE:** `identity_id` can only be specified when `authentication_type` is `identityBased`. It must be one of the `identity_ids` of the Iot Hub. If not specified when `authentication_type` is `identityBased`, System Assigned Managed Identity of the Iot Hub will be used.
+-> **NOTE:** `identity_id` can only be specified when `authentication_type` is `identityBased`. It must be one of the `identity_ids` of the Iot Hub. If `identity_id`is omitted when `authentication_type` is `identityBased`, then the System Assigned Managed Identity of the Iot Hub will be used.
 
-~> **NOTE:** System Assigned Managed Identity can only be used in an update because access to the endpoint cannot be granted before the creation is done. The extracted resources `azurerm_iothub_endpoint_*` can be used to create endpoints with System Assigned Managed Identity without the need of an update. 
+~> **NOTE:** An IoT Hub can only be updated to use the System Assigned Managed Identity for `endpoint` since it is not possible to grant access to the endpoint until after creation. The extracted resources `azurerm_iothub_endpoint_*` can be used to configure Endpoint with the IoT Hub's System Assigned Managed Identity without the need for an update.
```

Comment:
```suggestion
~> **NOTE:** An IoT Hub can only be updated to use the System Assigned Managed Identity for `endpoint` since it is not possible to grant access to the endpoint until after creation. The extracted resources `azurerm_iothub_endpoint_*` can be used to configure Endpoints with the IoT Hub's System Assigned Managed Identity without the need for an update.
```